### PR TITLE
AB tests (Chi-Square and Bayes) for aggregated data

### PR DESCRIPTION
**Pull request by hideaki**
# Description
AB tests (Chi-Square and Bayes) for aggregated data

# Checklist
Make sure you have performed the following items before submitting this pull request.
If not, please describe the reason.  

- [x] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: tests/testthat/test_exp_bayes_ab.R [hunk 1]
```diff
@@ -1,4 +1,36 @@
...
+    group_by(grouping)
+  full_data <- full_data %>%
+    dplyr::mutate(converted_true=as.integer(access_count*click), converted_false=as.integer(access_count*(1-click))) %>%
+    tidyr::gather(key="conver ted", value="count col", converted_true, converted_false) %>%
+    dplyr::mutate(`conver ted` = `conver ted`=="converted_true")
+  full_data_logical_ab <- full_data %>% dplyr::mutate(`gro up` = `gro up` == "a")
+
+  # revert A/B - character case
+  model_df <- exp_bayes_ab(full_data, `conver ted`, `gro up`, `count col`, prior_mean=0.1, prior_sd=0.01)
+  res <- model_df %>% tidy_rowwise(model, type = "summary", pretty.name = TRUE)
```
Comment (kei51e) [inline tests/testthat/test_exp_bayes_ab.R:27]: Check the output values.

**Review by kei51e — COMMENTED** [review]
Test 0 ro 1 row cases.

File: R/do_bayes_ab.R [hunk 2]
```diff
@@ -161,6 +161,19 @@ exp_bayes_ab <- function(df, converted, a_b_identifier, count = NULL, prior_mean
   }
 }
 
+exp_bayes_ab_aggregated <- function(df, a_b_identifier, conversion_rate, count, prior_mean = NULL, prior_sd = NULL, type = "model", revert_ab = FALSE, seed = 1, ...){
```
[the remark this answers is not in the document]
Comment (hideaki) [reply R/do_bayes_ab.R:164]: Done.

File: tests/testthat/test_exp_bayes_ab.R (hunk 1 above)
Comment (hideaki) [reply tests/testthat/test_exp_bayes_ab.R:27]: Done.

**Pull request merged**